worker/db.go

```diff
@@ -0,0 +1,48 @@
+package worker
+
+import (
+	"fmt"
+	"os"
+
+	"gorm.io/driver/postgres"
+	"gorm.io/gorm"
+	"github.com/mini-maxit/worker/utils"
+)
+
+// Connect to the database using GORM
+func connectToDatabase() *gorm.DB {
```

Comment:
Create a config struct and parse environment there, instead of during connection